src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -623,41 +623,53 @@ namespace shogun
 		}
 
 		/**
-		 * Compute the eigenvalues and eigenvectors of a symmetric matrix.
+		 * Compute the top-k eigenvalues and eigenvectors of a symmetric matrix.
```

Comment:
Usually, solvers that compute few eigenvalues are slower than those computing all of them.
Are two different solvers used when k<n and k==n?
If so, should be mentioned here